app/assets/javascripts/global-bar-init.js

```diff
@@ -0,0 +1,56 @@
+//= require libs/GlobalBarHelper.js
+
+'use strict'
+window.GOVUK = window.GOVUK || {}
+
+// Bump this if you are releasing a major change to the banner
+// This will reset the view count so all users will see the banner, even if previously seen
+var BANNER_VERSION = 3;
+
+var globalBarInit = {
+  getBannerVersion: function() {
+    return BANNER_VERSION
+  },
+
+  blacklistedUrl: function() {
+    return /^\/register-to-vote|^\/done|^\/brexit|^\/get-ready-brexit-check/.test(window.location.pathname)
```

Comment:
This would be easier to read if it was split out into an array of paths, can probably then do something like `.test(new RegExp(paths.join("|")))`